eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -47,11 +47,12 @@ steps:
 - task: MicroBuildSwixPlugin@1
   displayName: "Install Swix Plugin"
 
-- task: ms-vseng.MicroBuildTasks.965C8DC6-1483-45C9-B384-5AC75DA1F1A4.MicroBuildOptProfPlugin@4
+- task: ms-vseng.MicroBuildTasks.965C8DC6-1483-45C9-B384-5AC75DA1F1A4.MicroBuildOptProfPlugin@6
   displayName: 'OptProfV2:  install the plugin'
   inputs:
     getDropNameByDrop: true
     optimizationInputsDropNamePrefix: OptimizationInputs/$(System.TeamProject)/$(Build.Repository.Name)
+    AccessToken: $(System.AccessToken)
```

Comment:
@zivkan are we good to merge now with these changes?

Reply:
yes, I'll be happy for this to be merged ASAP.  Please make sure that the release pipeline that copies the artifacts is setup to maintain compatibility with the current/old build pipeline, and is not still using the temporary copy path.

This specific change, changing the version of the optprof pugin and using the access token, could have been done in the other branch regarding moving to the different agent pool. But there's no harm in doing it here, and we already have a green build, so I say go for it.

Reply:
Build 5.10.0.7212 was already tested with the old process, so I'm confident that backwards compatibility is fine. I removed the temporary paths 2 days ago.

I'll monitor *this* PR's release since it's the first true test without the Build pipeline having copied those same artifacts.